src/pages/places/[placeId]/availability.test.js

```diff
@@ -90,8 +90,8 @@ test('I can save a status with a reason', async () => {
 
   await waitForFetch(`/places/${page.router.query.placeId}/status`);
 
-  // 3rd API call, [url, payload] tuple
-  expect(fetch.mock.calls[3][1].body).toEqual(
+  // 4th API call, [url, payload] tuple
```

Comment:
Not sure I understand all those comments, I don't feel they provide much information that the code itself doesn't and they force you to have to keep both in sync

Reply:
@Anahkiasen yes you're right, I removed them 👍